src/services/codefixes/importFixes.ts

```diff
@@ -3,6 +3,7 @@ import {
     AnyImportOrRequireStatement,
     AnyImportSyntax,
     arrayFrom,
+    type BindingElement,

```

Comment:
Are the type modifiers intentional, or just leftover from an editor setting?

Shouldn't matter too much now given dprint sorts these and I forced it in the settings.json template, but it is _technically_ inconsistent 😄

Reply:
Left over, will fix